client-lib/src/main/java/com/ibm/ws/repository/resources/writeable/RepositoryResourceWritable.java

```diff
@@ -336,4 +337,11 @@
      */
     public String getAssetURL();
 
+    /**
+     * Returns the Last updated on.
+     *
+     * @return Calendar - the last updated on.
+     */
+    public Calendar getLastUpdatedOn();
```

Comment:
This needs a slightly better comment. The comment should usually expand on the information in the method's name.

In this case, the method is returning the date and time when the resource was last updated.

Also, you don't need to give the return type in the comment (even though the method above does...)